core_data_modules/cleaners/codes.py

```diff
@@ -2,4 +2,10 @@ class Codes(object):
     male = "male"
     female = "female"
 
+    yes = "yes"
```

Comment:
Consider either enumerations or YES

Reply:
Will change to upper case.

Enumerations are a thing in Python, but were only added in 3.4. I'm therefore not using them because I didn't think we wanted to use a backports library for 2.7